test/with_dummyserver/test_connectionpool.py

```diff
@@ -372,7 +392,7 @@ def test_bad_connect(self):
             pool.request('GET', '/', retries=5)
             self.fail("should raise timeout exception here")
         except MaxRetryError as e:
-            self.assertTrue(isinstance(e.reason, ProtocolError), e.reason)
+            self.assertEqual(type(e.reason), NewConnectionError)
```

Comment:
Can we use [`assertIsInstance`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsInstance) here? (Ugh I hate unittest).